host/tools/build/Microsoft.Azure.Functions.DotnetIsolatedNativeHost.nuspec

```diff
@@ -4,7 +4,7 @@
     <id>Microsoft.Azure.Functions.DotNetIsolatedNativeHost</id>
     <title>Microsoft Azure Functions dotnet-isolated native host</title>
     <tags>dotnet-isolated azure-functions azure</tags>
-    <version>1.0.6</version>
+    <version>1.0.7-preview1</version>
```

Comment:
Will remove the preview suffix before merging (after testing on both platforms are done)

Reply:
Validated on Linux as well.